src/einsteinpy/tests/test_symbolic/test_vaccum_metrics.py

```diff
@@ -0,0 +1,11 @@
+import numpy as np
+import sympy
+
+from einsteinpy.symbolic import MetricTensor, vaccum_metrics
+
+
+def schwarzschild_metric():
+    symbolstr = "t r theta phi"
+    syms = sympy.symbols(symbolstr)
+    sch = vaccum_metrics.SchwarzschildMetric(syms)
```

Comment:
You defined `SchwarzschildMetric()` function to take a string as an argument, but here you are passing a list of sympy expressions. Kindly take care of these mistakes. Anyways, you already have defined a function for schwarzschild metric in the `vaccum_metrics.py`, why are you defining it here again.

You may remove it.

Reply:
Also spelling of vacuum is wrong everywhere, its "vacuum" not "vaccum". It's a common pitfall. Been there, done that :D